StortSpelProjekt/Game/src/Gamefiles/GameGUI.cpp

```diff
@@ -33,6 +34,12 @@ void GameGUI::Update(double dt, Scene* scene)
 		}
 	}
 
+	if (scene->GetName() == "OptionScene")
+	{
+		Entity* entity = scene->GetEntity("volume");
+		entity->GetComponent<component::GUI2DComponent>()->GetTextManager()->SetText(Option::GetInstance().GetVariable("f_volume"), "volume");
```

Comment:
I suppose this is to make the text "volume" appear? It does not show up for me
![image](https://user-images.githubusercontent.com/38034730/98663681-aef7eb00-2349-11eb-9c1b-745f04481d0e.png)

Reply:
There is no text marking that as volume, i was just so used to it being volume i didn't question it